src/routes/+page.svelte

```diff
@@ -191,6 +191,7 @@
             target="_blank">AUT City Campus</a
           > is within minutes of the central business district, has excellent public transport and accommodation
           options.
+          Coordinates: -36.853388, 174.765877
```

Comment:
Should we do this as a Google Maps link?

Like this.

```suggestion
          Coordinates: [-36.853388, 174.765877](https://www.google.com/maps/place//@-36.8532934,174.7652698,139m)
```

Reply:
I didn't know whether Google Maps was FOSS4G enough. I've also been suggested to put a street address instead of coordinates to make it more easily understandable by anyone, but maybe the link to gmaps is easy enough?